verify_readiness.py

```
@@ -97,15 +111,16 @@ def check_prepush_checklist():
     
     # 1. Frontend Formatting Check
     try:
-        subprocess.check_output(["npm", "--prefix", "frontend", "run", "format:check"], stderr=subprocess.STDOUT)
+        npm_exe = "npm.cmd" if sys.platform == "win32" else "npm"
+        subprocess.check_output([npm_exe, "--prefix", "frontend", "run", "format:check"], stderr=subprocess.STDOUT)
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)